packages/fluentui/e2e/tests/popupIframeDismiss-example.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Popup, popupContentClassName, Button } from '@fluentui/react-northstar';
+
+export const selectors = {
```

Comment:
you could rename this suite/example to `iframeDismiss` and then just add all the components that support iframe dismiss here

That way you can write separate tests for each component that supports iframe dismiss in the same suite. 

This test should be good enough until we diverge in usage of the `useOnIframeFocus` hook, so it's not blocking this PR, just a suggestion.